asf_tools/composite.py

```diff
@@ -163,58 +164,57 @@ def make_composite(outfile, infiles=None, path=None, pol=None, resolution=None):
         resolution = max([info['geoTransform'][1] for info in raster_info.values()])
 
     # resample infiles to maximum resolution & common UTM zone
-    raster_info = reproject_to_target(raster_info, target_epsg_code=target_epsg_code, target_resolution=resolution)
+    with TemporaryDirectory() as temp_dir:
```

Comment:
I'm annoyed by how long-lived this context manager is, and by how deep the indenting gets.  Possibly resolved by implementing a function for handling a particular raster inside the later `for` loop:
```
for raster, info in raster_info.items():
    process_raster(...)
```